Sources/ProjectDescription/TargetAction.swift

```diff
@@ -36,6 +36,9 @@ public struct TargetAction: Codable, Equatable {
 
     /// List of output filelist paths
     public let outputFileListPaths: [Path]
+    
+    /// Whether to skip running this script in incremental builds, if nothing has changed
+    public let basedOnDependencyAnalysis: Bool?
```

Comment:
As per Pedro's suggestion, I made this flag optional so that by default, whatever `XcodeProj` configures for this will be considered.

Reply:
Looks good